Intersect.Server.Framework/Items/MapItemSource.cs

```diff
@@ -0,0 +1,12 @@
+using Intersect.Server.Framework.Maps;
+
+namespace Intersect.Server.Framework.Items;
+
+public partial class MapItemSource: IItemSource
+{
+    public Guid Id { get; init; }
+    public ItemSourceType SourceType => ItemSourceType.Map;
+    public WeakReference<IMapInstance> MapInstanceReference { get; init; }
+    public Guid MapInstanceId { get; init; }
+    public Guid MapControllerId { get; init; }
```

Comment:
One of these two should be stored in `Id` -- I'm fine with `Id` and `InstanceId` or `Id` and `DescriptorId`.